docs/modules/ROOT/pages/vshn-managed/nextcloud/collabora.adoc

```diff
@@ -14,7 +14,8 @@ spec:
     service:
       collabora:
         enabled: true
-        fqdn: my-collabora.example.com # <1>
+        fqdn:
+        - my-collabora.example.com # <1>
```

Comment:
collabora does not allow multiple FQDNs, therefore it's a string

Reply:
ah yes, good catch